packages/flutter_background_service/README.md

```diff
@@ -15,6 +14,36 @@ A flutter plugin for execute dart code in background.
 > - in android/build.gradle ```classpath 'com.android.tools.build:gradle:7.1.2'```
 > - in android/gradle/wrapper/gradle-wrapper.properties ```distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip```
 
+### Configuration required for Foreground Services on Android 14 (SDK 34)
+
+Applications that target SDK 34 and use foreground services need to include some additional configuration to declare the type of foreground service they use:
+
+* Determine the type of foreground service your app requires by consulting [the documentation](https://developer.android.com/about/versions/14/changes/fgs-types-required)
+* Update your `android/app/build.gradle` file to set the manifest placeholder to the value to use for `android:foregroundServiceType`:
+
+```gradle
+android {
+    ...
+
+    defaultConfig {
+        ...
+        manifestPlaceholders['foregroundServiceType'] = 'connectedDevice'
```

Comment:
why was `connectedDevice` chosen? (I tried to check the [docs](https://developer.android.com/about/versions/14/changes/fgs-types-required#connected-device) but I can't understand why)

Reply:
`connectedDevice` was chosen arbitrarily to illustrate the type of value this option expects - library consumers would be expected to select a value that fits their use case (see the first bullet point 😄).

I'll update the README to make it clearer that users should replace this value with one of the available options

Reply:
oh, thanks!